drivers/ieee802154/Kconfig.nrf5

```diff
@@ -78,6 +78,28 @@ config IEEE802154_NRF5_DELAY_TRX_ACC
 	  Accuracy of the clock used for scheduling radio delayed operations (delayed transmission
 	  or delayed reception), in ppm.
 
+config IEEE802154_NRF5_CALLBACK_TIMEOUT_MS
```

Comment:
I'd opt for `CALLOUT` instead of `CALLBACK`.

Reply:
Fixed